xdsl/pattern_rewriter.py

```diff
@@ -313,6 +308,18 @@ def move_region_contents_to_new_regions(self, region: Region) -> Region:
             )
         return Rewriter.move_region_contents_to_new_regions(region)
 
+    def iter_affected_ops(self) -> Generator[Operation, None, None]:
```

Comment:
Nit: I would suggest to use `Iterable[Operation]` instead, of `Generator[Operation, None, None]`

Reply:
- [ ] Iterable[Operation]